go.mod

```diff
@@ -29,72 +29,71 @@ require (
 	google.golang.org/grpc v1.36.0
 	google.golang.org/protobuf v1.26.0
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
-	k8s.io/api v0.21.0-beta.1
-	k8s.io/apimachinery v0.21.0-beta.1
-	k8s.io/apiserver v0.21.0-beta.1
+	k8s.io/api v0.21.0
+	k8s.io/apimachinery v0.21.0
+	k8s.io/apiserver v0.21.0
 	k8s.io/client-go v1.5.2
```

Comment:
Should we update this to v0.21.0? v1.5.0 is super ancient...

Reply:
Further down we have
```
replace k8s.io/client-go => k8s.io/client-go v0.21.0
```
i.e. in practice we are already using 0.21.0 as part of this PR.

I've fixed this line to reflect that.